st2common/st2common/models/db/trigger.py

```diff
@@ -77,6 +77,8 @@ class TriggerDB(stormbase.StormBaseDB, stormbase.ContentPackResourceMixin,
 
     name = me.StringField(required=True)
     pack = me.StringField(required=True, unique_with='name')
+    enabled = me.BooleanField(required=True, default=True,
```

Comment:
I know it's WIP, but before we merge it we should also document what "disabled trigger" implies (what kind of behavior, etc.).